examples/platforms/posix/radio.c

```diff
@@ -308,16 +309,24 @@ void otPlatRadioSetPromiscuous(bool aEnable)
 void posixRadioInit(void)
 {
     struct sockaddr_in sockaddr;
+    char *offset;
     memset(&sockaddr, 0, sizeof(sockaddr));
     sockaddr.sin_family = AF_INET;
 
+    offset = getenv("PORT_OFFSET");
+
+    if (offset)
+    {
+        sPortOffset = strtol(offset, NULL, 0) * WELLKNOWN_NODE_ID;
```

Comment:
Note: If no conversion could be performed, 0 is returned and the global variable errno is set to EINVAL (the last feature is not portable across all platforms). might want to at least print an error, s.t. there is a feedback to the user. Additionally, range checking on `sPortOffset` would be appropriate in production code; here you can go either way